### PR TITLE
295396 update install adapter section

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/main/shared-content/installation/install-using-docker.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+---
+uid: InstallUsingDocker
+---
+
+# Installation using Docker
+
+Docker is a set of tools that you can use on Linux to manage application deployments. This topic provides examples of how to create a Docker container with the adapter.
+
+**Note:** If you want to use Docker, you must be familiar with the underlying technology and have determined that it is appropriate for your planned use of the adapter. Docker is not required to use the adapter.
```
Comment (KNyce71) [inline content/main/shared-content/installation/install-using-docker.md:9]:
```suggestion
**Note:** If you want to use Docker, OSIsoft recommends that you be both familiar with the underlying technology and have determined that it is appropriate for your planned use of the adapter. Docker is not required to use the adapter.
```

Comment (KNyce71) [reply content/main/shared-content/installation/install-using-docker.md:9]: Just a minor suggestion.....feel free to ignore if it isn't a good one.

**Review by KNyce71 — COMMENTED** [review]
I've included just one minor suggestion.

**Review by rmuolic — DISMISSED** [review]

File: content/main/shared-content/installation/install-using-docker.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+---
+uid: InstallUsingDocker
+---
+
+# Installation using Docker
+
+Docker is a set of tools that you can use on Linux to manage application deployments. This topic provides examples of how to create a Docker container with the adapter.
+
+**Note:** Installation using Docker in only recommended for users proficient with Docker. Docker is not required to use the adapter.
```
Comment (rmuolic) [inline content/main/shared-content/installation/install-using-docker.md:9]:
```suggestion
**Note:** The use of Docker is only recommended if your environment requires it. Only users proficient with Docker should use it to install the adapter. Docker is not required to use the adapter.
```

**Review by rmuolic — APPROVED** [review]

**Pull request merged**